## [Analysis of Medicare Dialysis Facilities data from 2017 to 2020](https://data.cms.gov/quality-of-care/medicare-dialysis-facilities)

The purpose of this notebook is to ingest data from the Center for Medicare and Medicaid Services of dialysis facilities for 2017 through 2020 and answer research questions about characteristics of End-Stage Renal Disease (ESRD) patients, medicare dialysis facilities, and standardized readmission rates (SRR), an indicator of quality of care in these facilities.

The key research questions to answer are:
* Has the number of patients in dialysis facilities increased between 2017 and 2020?
* Is there an association between the number of patients in dialysis facilities and the share of patients sharing from comorbidities such as diabetes?
* Do patients who received Pre-ESRD Nephrologist Care, on average, have lower SRR compared to those that did not?

In [0]:
# The data for the analysis is saved in an AWS S3 object and is ingested below.
df = spark.read.format("csv") \
  .option("inferSchema", "true") \
  .option("header", "true") \
  .option("sep", ",") \
  .load("s3://medicare-dialysis-facilities/Medicare_Dialysis_Facility_Data_2022.csv")

# All features are string type except the CMS Certification Number (CCN), ESRD Network, and the measure value or score of every facility. 

In [0]:
# Display the first five rows of the data frame.
df.limit(5).toPandas()

state 
 CCN 
 Provider_Name 
 city 
 Ownership_Type 
 ESRD_Network 
 NPI 
 Chain 
 Modality 
 Alternate_CCNs 
 Measure 
 Measure_Score 
 year 
 Measure_ID 
 
 
 
 
 0 
 AL 
 12306 
 CHILDRENS HOSPITAL OF ALABAMA ESRD 
 BIRMINGHAM 
 Non-profit 
 8 
 1720166085 
 INDEPENDENT 
 Hemodialysis and Peritoneal Dialysis 
 012306,013300 
 F: COVID - Number of Medicare dialysis patient... 
 12.0 
 Quarter 2, 2020 
 allmcpatq2_f 
 
 
 1 
 AL 
 12306 
 CHILDRENS HOSPITAL OF ALABAMA ESRD 
 BIRMINGHAM 
 Non-profit 
 8 
 1720166085 
 INDEPENDENT 
 Hemodialysis and Peritoneal Dialysis 
 012306,013300 
 F: COVID - Number of patients first identified... 
 0.0 
 Quarter 2, 2020 
 allmcFcovpatq2_f 
 
 
 2 
 AL 
 12306 
 CHILDRENS HOSPITAL OF ALABAMA ESRD 
 BIRMINGHAM 
 Non-profit 
 8 
 1720166085 
 INDEPENDENT 
 Hemodialysis and Peritoneal Dialysis 
 012306,013300 
 F: COVID - % of Medicare patients first identi... 
 0.0 
 Quarter 2, 2020 
 allmcFcovpatPq2_f 
 
 
 3 
 AL 
 12306 
 CHILDRENS HOSPITAL OF ALABAMA ESRD 
 BIRMINGHAM 
 Non-profit 
 8 
 1720166085 
 INDEPENDENT 
 Hemodialysis and Peritoneal Dialysis 
 012306,013300 
 F: COVID - Number of patients ever identified ... 
 0.0 
 Quarter 2, 2020 
 allmcEcovpatq2_f 
 
 
 4 
 AL 
 12306 
 CHILDRENS HOSPITAL OF ALABAMA ESRD 
 BIRMINGHAM 
 Non-profit 
 8 
 1720166085 
 INDEPENDENT 
 Hemodialysis and Peritoneal Dialysis 
 012306,013300 
 F: COVID - % of Medicare patients ever identif... 
 0.0 
 Quarter 2, 2020 
 allmcEcovpatPq2_f

In [0]:
# Understand the data more by determining the shape of the data frame
# The data frame contains 10,821,385 rows and 14 columns
df.count(), len(df.columns)

Out[21]: (10821385, 14)

In [0]:
# Most of the dialysis facilities are for-profit institutions.
df.groupBy('Ownership_Type').count().show()

+--------------+-------+
|Ownership_Type|  count|
+--------------+-------+
|    For Profit|9624215|
|    Non-profit|1193248|
|   Unavailable|   3921|
|          null|      1|
+--------------+-------+



In [0]:
# The most common modality of dialysis is hemodialysis and peritoneal dialysis and the least frequent type of peritoneal dialysis
df.groupBy('Modality').count().sort("count").toPandas()

Modality 
 count 
 
 
 
 
 0 
 None 
 1 
 
 
 1 
 Unavailable 
 88161 
 
 
 2 
 Peritoneal Dialysis 
 144120 
 
 
 3 
 Hemodialysis 
 4810913 
 
 
 4 
 Hemodialysis and Peritoneal Dialysis 
 5778190

In [0]:
# The following are the tabulations of the `year` feature. Some measures were reported by quarter in 2020 and therefore, have data listed separately from the 2020 year value.
df.groupBy('year').count().sort("count").toPandas()

year 
 count 
 
 
 
 
 0 
 None 
 1 
 
 
 1 
 n/a 
 54619 
 
 
 2 
 Quarter 4, 2020 
 87857 
 
 
 3 
 Quarter 3, 2020 
 90557 
 
 
 4 
 Quarter 2, 2020 
 92199 
 
 
 5 
 Quarter 1, 2020 
 97890 
 
 
 6 
 2017-19 
 113469 
 
 
 7 
 2017-20 
 521687 
 
 
 8 
 2020 
 2286892 
 
 
 9 
 2017 
 2421398 
 
 
 10 
 2018 
 2509600 
 
 
 11 
 2019 
 2545216

In [0]:
# Examine describe statistics of key variables

In [0]:
# Create a temporary view of the data for writing SQL queries
# query the data below.
df.createOrReplaceTempView("medicare_dialysis_facilities")

In [0]:
%sql

select * from `medicare_dialysis_facilities` LIMIT 5;

state CCN Provider_Name city Ownership_Type ESRD_Network NPI Chain Modality Alternate_CCNs Measure Measure_Score year Measure_ID AL 12306 CHILDRENS HOSPITAL OF ALABAMA ESRD BIRMINGHAM Non-profit 8 1720166085 INDEPENDENT Hemodialysis and Peritoneal Dialysis 012306,013300 F: COVID - Number of Medicare dialysis patients, 2020-Q2 12.0 Quarter 2, 2020 allmcpatq2_f AL 12306 CHILDRENS HOSPITAL OF ALABAMA ESRD BIRMINGHAM Non-profit 8 1720166085 INDEPENDENT Hemodialysis and Peritoneal Dialysis 012306,013300 F: COVID - Number of patients first identified with COVID, 2020-Q2 0.0 Quarter 2, 2020 allmcFcovpatq2_f AL 12306 CHILDRENS HOSPITAL OF ALABAMA ESRD BIRMINGHAM Non-profit 8 1720166085 INDEPENDENT Hemodialysis and Peritoneal Dialysis 012306,013300 F: COVID - % of Medicare patients first identified with COVID, 2020-Q2 0.0 Quarter 2, 2020 allmcFcovpatPq2_f AL 12306 CHILDRENS HOSPITAL OF ALABAMA ESRD BIRMINGHAM Non-profit 8 1720166085 INDEPENDENT Hemodialysis and Peritoneal Dialysis 012306,013300 F: COVID - Number of patients ever identified with COVID (prior to or during current quarter), 2020-Q2 0.0 Quarter 2, 2020 allmcEcovpatq2_f AL 12306 CHILDRENS HOSPITAL OF ALABAMA ESRD BIRMINGHAM Non-profit 8 1720166085 INDEPENDENT Hemodialysis and Peritoneal Dialysis 012306,013300 F: COVID - % of Medicare patients ever identified with COVID, 2020-Q2 0.0 Quarter 2, 2020 allmcEcovpatPq2_f

In [0]:
%sql

SELECT Measure,
       ROUND(COUNT(Measure_Score),0) AS cnt_value,
       ROUND(AVG(Measure_Score),1) AS avg_value,
       ROUND(MIN(Measure_Score),1) AS min_value,
       ROUND(MAX(Measure_Score),1) AS max_value,
       ROUND(STD(Measure_Score),1) AS std_value
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty1_f","allcnty2_f","allcnty3_f","allcnty4_f",
         "dbprimy1_f","dbprimy2_f","dbprimy3_f","dbprimy4_f",
         "gnprimy1_f","gnprimy2_f","gnprimy3_f","gnprimy4_f",
         "htprimy1_f","htprimy2_f","htprimy3_f","htprimy4_f",
         "nephy12my1_f","nephy12my2_f","nephy12my3_f","nephy12my4_f",
         "srry1_f","srry2_f","srry3_f","srry4_f")
GROUP BY Measure
ORDER BY Measure DESC;

Measure cnt_value avg_value min_value max_value std_value F: SRR - Standardized Readmission Ratio, 2020 6241 1.0 0.0 2.9 0.4 F: SRR - Standardized Readmission Ratio, 2019 7169 1.0 0.0 2.5 0.3 F: SRR - Standardized Readmission Ratio, 2018 6893 1.0 0.0 3.8 0.3 F: SRR - Standardized Readmission Ratio, 2017 6632 1.0 0.0 2.3 0.3 F (AFS): Facility Information - Facility Information - Number of patients, 2020 7796 100.6 0.0 1561.0 66.2 F (AFS): Facility Information - Facility Information - Number of patients, 2019 7744 101.1 0.0 1400.0 67.4 F (AFS): Facility Information - Facility Information - Number of patients, 2018 7513 101.0 0.0 1388.0 67.9 F (AFS): Facility Information - Facility Information - Number of patients, 2017 7165 103.1 0.0 1455.0 68.7 F (2728): Incident Patients - Pre-ESRD Nephrologist Care Greater Than 12 months, 2020 3021 33.1 0.0 100.0 23.7 F (2728): Incident Patients - Pre-ESRD Nephrologist Care Greater Than 12 months, 2019 4789 32.0 0.0 100.0 22.8 F (2728): Incident Patients - Pre-ESRD Nephrologist Care Greater Than 12 months, 2018 4724 32.1 0.0 100.0 23.2 F (2728): Incident Patients - Pre-ESRD Nephrologist Care Greater Than 12 months, 2017 4633 30.9 0.0 100.0 22.9 F (2728): Incident Patients - Percent hypertension as primary cause ESRD, 2020 3021 29.0 0.0 100.0 17.8 F (2728): Incident Patients - Percent hypertension as primary cause ESRD, 2019 4789 29.7 0.0 100.0 17.5 F (2728): Incident Patients - Percent hypertension as primary cause ESRD, 2018 4724 30.3 0.0 100.0 17.7 F (2728): Incident Patients - Percent hypertension as primary cause ESRD, 2017 4633 30.5 0.0 100.0 18.0 F (2728): Incident Patients - Percent glomerulonephritis as primary cause of ESRD, 2020 3021 6.6 0.0 73.3 7.6 F (2728): Incident Patients - Percent glomerulonephritis as primary cause of ESRD, 2019 4789 6.4 0.0 64.3 7.1 F (2728): Incident Patients - Percent glomerulonephritis as primary cause of ESRD, 2018 4724 6.7 0.0 61.5 7.1 F (2728): Incident Patients - Percent glomerulonephritis as primary cause of ESRD, 2017 4633 6.8 0.0 69.2 7.3 F (2728): Incident Patients - Percent diabetes as primary cause ESRD, 2020 3021 49.0 0.0 100.0 18.2 F (2728): Incident Patients - Percent diabetes as primary cause ESRD, 2019 4789 48.4 0.0 100.0 17.5 F (2728): Incident Patients - Percent diabetes as primary cause ESRD, 2018 4724 49.0 0.0 100.0 17.4 F (2728): Incident Patients - Percent diabetes as primary cause ESRD, 2017 4633 48.7 0.0 100.0 17.1

Overall, it appears that on average the SRR has not changed significantly between 2017 and 2020. However, the average SRR among all facilities has increased from 2019 but was lower compared to the highest on 2018. The SRR is a measure of quality because it evaluates total observed readmissions associated with a facility to the expected readmissions given the characteristics of the patients served in that facility. A lower SRR is evidence that fewer readmissions were necessary compared to the expected number and may signal that the quality of care was sufficiently high as to not require readmission into the facility or hospital. However, additional contextual factors such as patient comorbidities and care taken after a dialysis procedure may influence the rate at which patients are readmitted. The number of patients, on average, has remained at around 100. On average, the share of incident patients who received pre-ESRD nephrologist care for more than 12 months in creased from 31% in 2017 to 33% in 2020, and on average the share or incident patients who reported diabetes as the primary cause of ESRD increased from 48.7% in 2017 to 49% in 2020.

In [0]:
# Determine the number of null values in the dataframe.

from pyspark.sql.functions import *
display(df.select([count(when(col(c).isNull(),c)).alias(c) for c in df.columns]))

# This resource was used in identifying the number of null values in each column of the data frame: https://www.projectpro.io/recipes/get-null-count-of-each-column-of-dataframe-pyspark-databricks

state CCN Provider_Name city Ownership_Type ESRD_Network NPI Chain Modality Alternate_CCNs Measure Measure_Score year Measure_ID 1 1 1 1 1 1 1 1 1 1 1 1 1 1

The map below shows the states with the highest number of dialysis patients, including California, Texas, Florida, and New York. These are some of the most populated states, therefore, it may be expected that overall they will have a higher number of patients. The count of patients can be compared to the population size for the age group of dialysis patients in the dataset, which typically includes individuals above age 60.

In the following cell, the percentage growth in the number of dialysis patients by state is computed. States such as Utah, Maine, South Dakota, and Arizona account for the four highest states with the largest growth in the number of dialysis patients.

In response to the first research question of growth in the number of patients, on average, the number of patients is stable at around 100, in 2017 and 2020. However, some states have significantly higher growth rates in the number of dialysis patients compared to other states. It is imperative for state health agencies to understand the reasons behind the growth in the number of dialysis patients and implement preventative policies that will decrease the risk of ESRD for elderly patients.

In [0]:
%sql

SELECT state,
       SUM(Measure_Score) AS `Number of patients in 2020`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty4_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `Number of patients in 2020` DESC;

state Number of patients in 2020 CA 104732.0 TX 79563.0 FL 52368.0 NY 45855.0 IL 32422.0 GA 32416.0 PA 28314.0 OH 27780.0 NC 27472.0 MI 22712.0 NJ 21030.0 VA 19389.0 MD 17683.0 TN 17370.0 AZ 16781.0 LA 15408.0 SC 15034.0 IN 14971.0 AL 14586.0 MO 13138.0 WA 12083.0 MS 10916.0 WI 10511.0 MA 9834.0 KY 9295.0 OK 8697.0 MN 7993.0 NV 7605.0 AR 7227.0 CO 7221.0 OR 7156.0 CT 6135.0 HI 5691.0 NM 5453.0 KS 4678.0 IA 4392.0 WV 3821.0 UT 3511.0 NE 2797.0 DE 2698.0 DC 2545.0 ID 2312.0 ME 1808.0 SD 1734.0 NH 1700.0 RI 1636.0 MT 1348.0 ND 1263.0 AK 931.0 WY 573.0 VT 505.0

In [0]:
%sql

WITH patients_2017(state,nbr_patients_17)
AS (SELECT state,
       SUM(Measure_Score) AS `nbr_patients`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty1_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `nbr_patients` DESC),

patients_2020(state,nbr_patients_20)
AS (SELECT state,
       SUM(Measure_Score) AS `nbr_patients`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty4_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `nbr_patients` DESC)

SELECT a.state,
       a.nbr_patients_17 AS `Number of patients in 2017`,
       b.nbr_patients_20 AS `Number of patients in 2020`,
       ROUND(100 * ((b.nbr_patients_20 - a.nbr_patients_17) / (a.nbr_patients_17)),1) AS `Percentage growth`
FROM patients_2017 AS a
  FULL JOIN patients_2020 AS b
    ON a.state = b.state
ORDER BY `Percentage growth` DESC
LIMIT 10;

state Number of patients in 2017 Number of patients in 2020 Percentage growth UT 2910.0 3511.0 20.7 ME 1541.0 1808.0 17.3 SD 1500.0 1734.0 15.6 AZ 14695.0 16781.0 14.2 AK 817.0 931.0 14.0 NV 6714.0 7605.0 13.3 HI 5034.0 5691.0 13.1 NM 4889.0 5453.0 11.5 AR 6510.0 7227.0 11.0 NC 24813.0 27472.0 10.7

Based on the scatter plot below, it appears that there is a moderately negative relationship between the growth rate in the average share of patients whose primary cause of ESRD is diabetes and the growth in the number of patients. In other words, on average, in states where the average growth in the share of incidence patients whose primary cause of ESRD is diabetes, the growth rate in the number of patients tends to be lower. This highlights that there may be more reasons that explain the growth in the number of dialysis patient beyond diabetes as a comorbidity.

In [0]:
%sql

WITH patients_2017(state,nbr_patients_17)
AS (SELECT state,
       SUM(Measure_Score) AS `nbr_patients`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty1_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `nbr_patients` DESC),

patients_2020(state,nbr_patients_20)
AS (SELECT state,
       SUM(Measure_Score) AS `nbr_patients`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("allcnty4_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `nbr_patients` DESC),

diabetes_2020(state,diabetes_20)
AS (SELECT state,
       AVG(Measure_Score) AS `diabetes_20`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("dbprimy4_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `diabetes_20` DESC),

diabetes_2017(state,diabetes_17)
AS (SELECT state,
       AVG(Measure_Score) AS `diabetes_17`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("dbprimy1_f") AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
GROUP BY state
ORDER BY `diabetes_17` DESC)

SELECT a.state,
       a.nbr_patients_17 AS `Number of patients in 2017`,
       b.nbr_patients_20 AS `Number of patients in 2020`,
       ROUND(100 * ((b.nbr_patients_20 - a.nbr_patients_17) / (a.nbr_patients_17)),1) AS `% growth in patients`,
       c.diabetes_17 AS `Average share of patients whose primary cause of ESRD is diabetes in 2017`,
       d.diabetes_20 AS `Average share of patients whose primary cause of ESRD is diabetes in 2020`,
       ROUND(100 * ((d.diabetes_20 - c.diabetes_17) / (c.diabetes_17)),1) AS `% growth in diabetes as primary cause of ESRD`      
FROM patients_2017 AS a
  FULL JOIN patients_2020 AS b
    ON a.state = b.state
  FULL JOIN diabetes_2017 AS c
    ON a.state = c.state
  FULL JOIN diabetes_2020 AS d
    ON a.state = d.state
ORDER BY `% growth in patients` DESC;

state Number of patients in 2017 Number of patients in 2020 % growth in patients Average share of patients whose primary cause of ESRD is diabetes in 2017 Average share of patients whose primary cause of ESRD is diabetes in 2020 % growth in diabetes as primary cause of ESRD UT 2910.0 3511.0 20.7 50.97444444444446 62.135454545454536 21.9 ME 1541.0 1808.0 17.3 50.40555555555556 55.728333333333325 10.6 SD 1500.0 1734.0 15.6 63.62125 47.9125 -24.7 AZ 14695.0 16781.0 14.2 57.41348314606743 58.169384615384615 1.3 AK 817.0 931.0 14.0 59.512 47.519999999999996 -20.2 NV 6714.0 7605.0 13.3 50.69926829268294 42.167500000000004 -16.8 HI 5034.0 5691.0 13.1 69.66238095238096 63.96421052631579 -8.2 NM 4889.0 5453.0 11.5 63.489696969696965 61.719047619047615 -2.8 AR 6510.0 7227.0 11.0 45.59276595744681 42.919090909090905 -5.9 NC 24813.0 27472.0 10.7 49.26986394557822 49.27642201834861 0.0 TX 72044.0 79563.0 10.4 56.41707589285712 56.414061433447095 0.0 WY 520.0 573.0 10.2 58.94666666666668 47.620000000000005 -19.2 MT 1225.0 1348.0 10.0 48.80777777777777 40.5975 -16.8 OR 6513.0 7156.0 9.9 52.503720930232554 51.34137931034482 -2.2 WI 9614.0 10511.0 9.3 44.42923076923077 44.893469387755104 1.0 NH 1559.0 1700.0 9.0 49.739285714285714 43.16888888888889 -13.2 DE 2482.0 2698.0 8.7 43.67071428571428 51.125 17.1 IN 13826.0 14971.0 8.3 48.24542857142857 48.82716981132075 1.2 CA 97139.0 104732.0 7.8 55.28465909090908 55.04199494949494 -0.4 ID 2145.0 2312.0 7.8 51.1925 51.751 1.1 FL 48668.0 52368.0 7.6 43.19512738853502 43.3104366812227 0.3 SC 13993.0 15034.0 7.4 48.66010638297872 48.07359374999999 -1.2 NE 2607.0 2797.0 7.3 48.66454545454547 55.05461538461539 13.1 MS 10191.0 10916.0 7.1 48.66857142857144 45.69857142857143 -6.1 MN 7474.0 7993.0 6.9 47.51755555555557 49.65681818181818 4.5 TN 16271.0 17370.0 6.8 41.8857627118644 40.473875 -3.4 CT 5767.0 6135.0 6.4 45.190769230769234 47.208571428571425 4.5 OK 8182.0 8697.0 6.3 53.06148148148146 54.86885714285716 3.4 RI 1551.0 1636.0 5.5 45.294166666666655 50.419999999999995 11.3 VA 18397.0 19389.0 5.4 45.045040650406534 48.62560606060605 7.9 WA 11482.0 12083.0 5.2 50.76408450704226 50.388392857142854 -0.7 GA 30918.0 32416.0 4.8 38.87375634517764 42.21827586206897 8.6 CO 6922.0 7221.0 4.3 50.47340909090909 47.40217391304348 -6.1 PA 27160.0 28314.0 4.2 46.915949999999974 44.669074074074075 -4.8 NY 44104.0 45855.0 4.0 46.80290043290041 44.65835106382978 -4.6 LA 14829.0 15408.0 3.9 44.44553398058252 46.88586956521739 5.5 AL 14053.0 14586.0 3.8 48.02890000000003 45.42325581395348 -5.4 MD 17059.0 17683.0 3.7 42.420097087378636 42.79673913043477 0.9 KY 8980.0 9295.0 3.5 51.981159420289856 52.72124999999998 1.4 ND 1222.0 1263.0 3.4 41.477777777777774 36.57 -11.8 MI 22327.0 22712.0 1.7 47.930927152317864 45.6690291262136 -4.7 IL 31894.0 32422.0 1.7 43.95648044692739 43.7189247311828 -0.5 MA 9667.0 9834.0 1.7 43.57076923076922 42.800625000000004 -1.8 OH 27342.0 27780.0 1.6 47.757740112994334 47.67724999999999 -0.2 MO 13009.0 13138.0 1.0 46.494615384615386 43.78678571428571 -5.8 WV 3788.0 3821.0 0.9 51.444074074074074 59.02000000000002 14.7 NJ 20999.0 21030.0 0.1 42.603221476510086 43.97804123711341 3.2 IA 4482.0 4392.0 -2.0 50.08193548387096 48.504000000000005 -3.2 KS 4780.0 4678.0 -2.1 44.73885714285715 44.46695652173913 -0.6 DC 2782.0 2545.0 -8.5 32.76714285714286 36.786 12.3 VT 675.0 505.0 -25.2 43.56666666666666 57.14 31.2

In [0]:
%sql

SELECT state,
       CCN,
       Provider_Name,
       Measure_Score
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("srry4_f") 
AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')
LIMIT 5;

state CCN Provider_Name Measure_Score AL 12500 FMC CAPITOL CITY 0.55 AL 12501 GADSDEN DIALYSIS 1.14 AL 12502 TUSCALOOSA UNIVERSITY DIALYSIS 1.24 AL 12505 PHYSICIANS CHOICE DIALYSIS-MONTGOMERY 1.03 AL 12506 DOTHAN DIALYSIS 1.25

In [0]:
%sql

WITH pre_esrd(state,CCN,Provider_Name,Measure_Score,`Received pre-ESRD care for 12+ months`)
AS (SELECT state,
       CCN,
       Provider_Name,
       Measure_Score,
       (CASE WHEN Measure_Score > 60 THEN "Yes" ELSE "No" END) AS `Received pre-ESRD care for 12+ months`
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("nephy12my4_f") 
AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI')),

srr(state,CCN,Provider_Name,SRR)
AS (SELECT state,
       CCN,
       Provider_Name,
       Measure_Score
FROM `medicare_dialysis_facilities`
WHERE Measure_ID IN("srry4_f") 
AND state NOT IN('AS','FM','GU','MH','MP','PR','PW','VI'))

SELECT a.`Received pre-ESRD care for 12+ months` AS `60+% of patients received pre-ESRD care`,
       a.state AS `State`,
       AVG(b.SRR) AS `Average Standardized Readmission Ratio`
FROM pre_esrd AS a
  FULL JOIN srr AS b
    ON a.CCN = b.CCN
WHERE a.`Received pre-ESRD care for 12+ months` IS NOT NULL
GROUP BY a.state,a.`Received pre-ESRD care for 12+ months`
ORDER BY state,`60+% of patients received pre-ESRD care`;

60+% of patients received pre-ESRD care State Average Standardized Readmission Ratio No AK 0.8175 No AL 1.0221052631578948 Yes AL 0.86 No AR 0.9012903225806452 Yes AR 1.09 No AZ 1.0226315789473683 Yes AZ 0.9762500000000001 No CA 1.0193641618497107 Yes CA 1.1035 No CO 0.8925 Yes CO 1.0466666666666666 No CT 0.9954545454545453 Yes CT 1.155 No DC 1.116 No DE 0.78 Yes DE 1.1033333333333333 No FL 1.054950980392157 Yes FL 1.035238095238095 No GA 0.9847826086956523 Yes GA 0.95 No HI 0.763846153846154 Yes HI 0.9950000000000001 No IA 1.0200000000000002 Yes IA 1.11 No ID 0.9187500000000001 Yes ID 0.925 No IL 1.0696052631578945 Yes IL 1.0342857142857143 No IN 0.9782222222222224 Yes IN 1.20125 No KS 1.0389473684210528 Yes KS 0.8649999999999999 No KY 1.003939393939394 Yes KY 0.882 No LA 1.0744444444444443 Yes LA 1.0000000000000002 No MA 1.115 Yes MA 1.0035714285714286 No MD 0.936315789473684 Yes MD 0.64 No ME 0.825 Yes ME 1.01 No MI 0.9920930232558138 Yes MI 0.9881249999999999 No MN 0.9506249999999999 Yes MN 0.9550000000000001 No MO 0.9588235294117644 Yes MO 0.6719999999999999 No MS 1.0202631578947368 Yes MS 1.486666666666667 No MT 0.7899999999999999 Yes MT null No NC 0.9312941176470588 Yes NC 0.9228571428571426 No ND 0.7883333333333332 No NE 1.0223076923076921 No NH 0.9285714285714286 Yes NH 0.78 No NJ 1.1215 Yes NJ 0.9753333333333334 No NM 0.831111111111111 Yes NM 0.405 No NV 1.059666666666667 Yes NV 1.19 No NY 1.050705128205129 Yes NY 1.050322580645161 No OH 1.0921875000000003 Yes OH 0.953 No OK 1.0926666666666667 Yes OK 1.0625 No OR 0.8072222222222222 Yes OR 0.8836363636363637 No PA 1.0004819277108432 Yes PA 0.9283333333333333 No RI 0.632 Yes RI 0.92 No SC 0.9792727272727272 Yes SC 1.1166666666666667 No SD 0.8724999999999999 Yes SD 1.3866666666666667 No TN 0.9806349206349206 Yes TN 1.0358333333333334 No TX 1.0063636363636368 Yes TX 1.0192307692307694 No UT 0.7310000000000001 Yes UT 1.18 No VA 0.9760377358490565 Yes VA 0.768 Yes VT 0.66 No WA 0.8819148936170214 Yes WA 0.92 No WI 0.9449999999999998 Yes WI 0.8827272727272728 No WV 0.9872222222222224 Yes WV 1.2100000000000002 No WY 1.275

As the chart above demonstrates, for 2020, on average, facilities for which 60% or more of patients received pre-ESRD nephrologist care for more than 12 months, had lower standardized readmission rates in states such as New Mexico. This is expected, as greater availability of specialized care prior and during the process of dialysis is expected to result in patients being better prepared to manage their condition and not have to be readmitted to a hospital or dialysis facility due to a dialysis procedure. However, in other states such as Mississippi, the SRR is significantly higher for patients who received more than 12 months of pre-ESRD care. Therefore, state health agencies can learn about their best practices and programs in states such as New Mexico, which have on average, lower readmission rates, compared to other states, particularly from facilities whose patients received 12 or more months of pre-ESRD nephrologist care.

The `Overview` notebook from Databricks was used as a reference in the initial steps of this analysis, particularly in the data ingestion commands and the creation of a temporary view for querying the dataset.